VideoLocker/src/main/java/org/edx/mobile/util/NetworkUtil.java

```diff
@@ -161,11 +161,11 @@ public static boolean isOnSocialDisabledNetwork(Context context){
 
     }
 
-    public static boolean isSocialFeatureFlagEnabled(Context context){
+    public static boolean isSocialFeatureFlagEnabled(Context context, Config config){
```

Comment:
This breaks `BaseFragmentActivityTest`, where the method called without any change in the parameters.